snc.sh

```diff
@@ -124,6 +124,15 @@ fi
 rm id_rsa_crc* || true
 ssh-keygen -N "" -f id_rsa_crc
 
+# Set NetworkManager DNS overlay file
+cat << EOF | sudo tee /etc/NetworkManager/dnsmasq.d/openshift.conf
+server=/crc.testing/192.168.126.1
+address=/apps-crc.testing/192.168.126.11
+EOF
+
+# Reload the NetworkManager to make DNS overlay effective
+sudo systemctl reload NetworkManager
+
```

Comment:
why only this specific change? This assumes there is already `/etc/NetworkManager/conf.d/openshift.conf`, and there is more pre-configuration needed (firewalld changes, libvirtd changes). I'm not saying I'm against it, but a more detailied rationale in the commit log would be useful.

Reply:
An alternative to doing the config ourselves would be to only test that the prerequisite are met (in other words, try to resolve `aaa.crc.testing` and error out if this is not using the DNS server we expect, try to connect to the libvirt port with nc, ...)

Reply:
@cfergeau I just want to have this part not other pre-requisites as of now since I mostly use the GCP where the pre-requisites are set except this one. I will just update the commit message as of now for it.